addon/locales/en/translations.js

```diff
@@ -76,7 +76,7 @@ export default {
     },
 
     'flexberry-maplayer': {
-      'opacity': 'Opacity'
+      'opacity': 'Visibility'
```

Comment:
why? may be need to rename property?

Reply:
I do not know why it is called opacity in leaflet, but I think that better to name properties according to their usage, and just to change caption. It is only caption

Reply:
ok